### DISCUSSION TITLE
channels.json not updated in real time

### DISCUSSION THREAD

**Discussion by Kabouik** [iptv-org/community#253] [Q&A] [answered]
I am using `https://iptv-org.github.io/iptv/channels.json` with `https://github.com/Roshan-R/termv` and I noticed that if I download a new copy of `channels.json`, it does not include the latest changes in `master` here. Just wondering how often is it updated?

**Comment by Dum4G** [comment, marked as answer]
Daily at 0:00 GMT (a bit later since updating process takes a while)

**Comment by Kabouik** [comment]
Thanks!
